ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -2926,3 +2926,39 @@ def test090_if_cancel_archive_order_no_order_suborder_analysis_will_archived(sel
             self.orders_page.filter_by_analysis_number(filter_text=analysis_no[i])
             self.assertEqual(len(self.order_page.result_table()) - 1, 0)
 
+    def test091_order_of_testunits_in_analysis_section(self) :
+        """
+        Ordering test units Approach: In case I put test plans and test units at the same time , the order of
+        the analysis section should be the test units of the test plans then the order test units
+
+        LIMS-7416
+        """
+
+        self.test_plan_api = TestPlanAPI()
+        payload, testplan = self.test_plan_api.create_completed_testplan_multiple_testunits()
+        testplan_name = (payload['testPlan']['text'])
+        articletype = payload['selectedArticles'][0]['text']
+        materialtype = (payload['materialType'][0]['text'])
+        testunit_name1 = (payload['testUnits'][0]['name'])
+        testunit_name2 = (payload['testUnits'][1]['name'])
+        self.order_page.create_new_order(material_type=materialtype, article=articletype, contact='',
+                                         test_plans=[testplan_name],
+                                         test_units=['new_test_unit'])
```

Comment:
you should select random test unit and create order with it, as if we used empty data base, '`new_test_unit`" won't be found

Reply:
done